tests/baselines/reference/nodeModulesImportAttributesTypeModeDeclarationEmit(module=node16).js

```diff
@@ -32,5 +32,5 @@ exports.b = null;
 
 //// [index.d.ts]
 export type LocalInterface = import("pkg", { with: { "resolution-mode": "require" } }).RequireInterface & import("pkg", { with: { "resolution-mode": "import" } }).ImportInterface;
-export declare const a: import("pkg").RequireInterface;
+export declare const a: import("pkg", { with: { "resolution-mode": "require" } }).RequireInterface;

```

Comment:
This is a behavior change, but, I think an okay one since we're trying to make it so that isolatedDeclaration doesn't have to reason about this at all.

Reply:
We usually elide them if they weren't needed because they matched the default mode for the file, yeah, but there's no harm (other than ascetic) in including them if they were written explicitly.